### ISSUE TITLE
Elevated circulating homovanillic acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6431]
**Preferred term label:** Elevated circulating homovanillic acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of homovanillic acid in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Elevated circulating catecholamine level HP:0003334; Abnormal circulating carboxylic acid concentration HP:0004354

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
